cluster-state-service/internal/features/_test/gucumbertest__.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2016-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Looks like you committed the gucumber `__.go` files. Can you remove these, as these should not be in source control.